react/week1/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import "./index.css";
+class todList extends Component {
+          state = {
+                    key1: { color: "green" },
+                    key2: { color: "red" },
+                    key3: { color: "blue" },
+          };
+          render() {
+                    return (
+                              <div className="wraper">
+                                        <h1>Todo List</h1>
+                                        <p style={this.state.key1}>
+                                                  {this.props.tList.key1}
+                                        </p>
+                                        <p style={this.state.key2}>
+                                                  {" "}
+                                                  {this.props.tList.key2}
+                                        </p>
+                                        <p style={this.state.key3}>
+                                                  {this.props.tList.key3}
+                                        </p>
+                              </div>
+                    );
+          }
```

Comment:
@tariqjavid overall it looks good!
There is just a couple things that need to be fixed:
- The `state` should be inside the `constructor`
- Instead of having the todos in an object, we can store them in an array and loop throgh them, fx:
```suggestion
    constructor(props){
        super(props)
        this.state={
            todos = [
                {
                    id: 1,
                    description: "Get out of bed",
                    deadline: "Wed Sep 13 2017",
                },
            ]
        }
    }
    return (
        <div>
            {this.state.todos.map(todo => (
                <li key={todo.id}>
                    {todo.description}, {todo.deadline}
                </li>
            ))}
        </div>
    );
```
And we can break the code even further and create a component `<Todo/>` and pass it the description and the deadline